sdk/containerregistry/azure-mgmt-containerregistry/azure/mgmt/containerregistry/v2019_06_01_preview/models/__init__.py

```diff
@@ -231,6 +251,9 @@
     UpdateTriggerPayloadType,
     SourceRegistryLoginMode,
     SecretObjectType,
+    TokenCertificateName,
+    TokenPasswordName,
+    Status,
```

Comment:
seems introduction of this enum, caused renaming existing "Status" model to "Status1".
I think we should figure out how to change rest api spec to make sure it's not renamed....